site/docs/components/container.mdx

```diff
@@ -1,11 +1,11 @@
 ---
-title: "Card"
-navTitle: "Card"
-summaryParagraph: Cards are a structural component to contain primary content.
-tags: ["Wells", "Panels", "Slats", "Rows", "List Rows", "Tiles", "Container", "Content area", "Box", "Rectangle"]
-githubLabels: ["component:card"]
+title: "Container"
+navTitle: "Container"
+summaryParagraph: Containers are a structural component to contain primary content (previously known as Card).
+tags: ["Cards", "Wells", "Panels", "Slats", "Rows", "List Rows", "Tiles", "Container", "Content area", "Box", "Rectangle"]
+githubLabels: ["component:container"]
```

Comment:
@lloydstubber Something I didn't consider is we may want to rename this tag in Github too.

Reply:
Yep, I'll see if I can rename it, otherwise will replace all card ones to container.